### PR TITLE
BYOQM -> Modu

### PR DESCRIPTION
**Pull request by FrederikRothe**
## Describe your changes
NUMBER 100 ⚡ ⚡ ⚡ 

Works and runs, also with cpd.

Total name change, everywhere BYOQM should have become modu.

If issues with runnining, please check your python env in poetry. Should also say modu, if not create a new one with python 3.10.

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] If it is a core feature, I have added thorough tests.
- [x] Have I requested and notified the boys for review of the pull-request?

### CODE REVIEW

File: .idea/.gitignore
Comment (SadCheeto) [inline .idea/.gitignore:1]: This should not be here

File: .idea/inspectionProfiles/profiles_settings.xml
Comment (SadCheeto) [inline .idea/inspectionProfiles/profiles_settings.xml:1]: Neither should this

File: .idea/misc.xml
Comment (SadCheeto) [inline .idea/misc.xml:1]: or this

File: .idea/modules.xml
Comment (SadCheeto) [inline .idea/modules.xml:1]: or this

File: .idea/tool.iml
Comment (SadCheeto) [inline .idea/tool.iml:1]: or this

File: .idea/vcs.xml
Comment (SadCheeto) [inline .idea/vcs.xml:1]: or this

**Review by SadCheeto — COMMENTED** [review]
Looks good overall. I will not approve this however, as there are 6 .idea files that should not be in this PR

**Review by toffernator — APPROVED** [review]
I have deleted the .idea files and changed `poetry run main` to `poetry run modu`

**Pull request merged**